### ISSUE TITLE
Impact of logging on Crossbar performance

### ISSUE THREAD

**Issue by om26er** [crossbario/crossbar#1927]
Logging seems to have quite a significant impact on the performance of Crossbar. We should consider tuning down the default (info) logging quite a bit.

Here is one such example https://github.com/crossbario/crossbar/issues/1899#issuecomment-991905161 where logging affects the startup performance of realms

**Comment by oberstet** [comment]
Does it still log large amounts of noise with log level "warn"? If not, why not switch to that? I mean, "warn" should be fine for regular operation .. and "info" actually log stuff which can savely be ignored any time ..

**Comment by om26er** [comment]
With log level "warn" is virtually logs nothing. I guess errors will show up. So I am fine with using that level by default as long as it's "recommended" :-)

**Comment by oberstet** [comment]
yeah, "warn" should be good enough for default level. it should only log things going wrong router/node side - not stuff going wrong at the app level, or infos about router/node status (as in stats)

eg a callee raising an exception would be logged at level "info", while a worker failing should log at level "warn". a callee client connection failing is only "info" .. whereas a rlink failing is probably "warn"

if above is how it's currently logging, I'm not sure .. but roughly should work like this ..
